bundles/org.eclipse.ui.workbench.texteditor/META-INF/MANIFEST.MF

```diff
@@ -8,21 +8,89 @@ Bundle-ActivationPolicy: lazy
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
 Export-Package: 
- org.eclipse.ui.contentassist,
- org.eclipse.ui.internal.texteditor;texteditor=split;mandatory:=texteditor;x-friends:="org.eclipse.ui.editors",
+ org.eclipse.ui.contentassist;uses:="org.eclipse.osgi.util,org.eclipse.swt.widgets,org.eclipse.jface.contentassist",
+ org.eclipse.ui.findandreplace;x-internal:=true,
+ org.eclipse.ui.internal.texteditor;texteditor=split;mandatory:=texteditor;x-friends:="org.eclipse.ui.editors,org.eclipse.ui.findandreplace",
```

Comment:
I think this cannot work as `org.eclipse.ui.findandreplace` is not a bundle, so you cannot declare it in `x-friends`. All the other changes in the manifest seem unnecessary to me.